app/javascript/stylesheets/application.css

```diff
@@ -8,31 +8,24 @@
     @apply text-lg md:text-xl xl:text-2xl;
   }
 
-  h1 {
+  .header-1 {
     @apply text-6xl md:text-7xl 2xl:text-8xl font-bold !important;
   }
 
-  h2 {
+  .header-2 {
     @apply text-4xl md:text-5xl 2xl:text-6xl font-medium !important;
   }
 
-  h3,
-  h4 {
+  .header-3-4 {
```

Comment:
WDYT about defining both?
`.header-3, .header-4`?

Not sure why not use `h3` instead of `h4`, avoiding using both elements as the same...